cmd/sops/subcommand/exec/exec.go

```diff
@@ -0,0 +1,100 @@
+package exec
+
+import (
+	"log"
```

Comment:
Instead of using the stdlib `log` package, can we switch to using `logrus` and importing the logrus config in sops?

i.e.
https://github.com/mozilla/sops/blob/14a22d7a7060a0fba06c5dd716898d47728f0fce/cmd/sops/subcommand/publish/publish.go#L15-L26